pilot/cmd/pilot-agent/main.go

```diff
@@ -476,9 +479,9 @@ func getDNSDomain(domain string) string {
 	return domain
 }
 
-// check if SDS UDS path and token path exist, if both exist, requests key/cert
-// using SDS instead of secret mount.
-func detectSds(controlPlaneBootstrap, controlPlaneAuthEnabled bool, udspath, trustworthyJWTPath string) (bool, string) {
+// detectSds checks if the UDS and JWT paths are present. The returned values are used by caller
+// to decide whether SDS is used for control plane.
+func detectSds(controlPlaneBootstrap bool, udspath, trustworthyJWTPath string) (bool, string) {
 	if !sdsEnabledVar.Get() {
 		return false, ""
 	}
```

Comment:
So are you removing the unix: prefix somehwere I didn't find ? 

My advice is again to just have a constant in Pilot with the value we use in the installer - it's obvious setting arbitrary values for the uds path is not tested, nor is it required at this point.